Sources/Plasma/NucleusLib/pnKeyedObject/plKey.h

```diff
@@ -83,7 +82,7 @@ class plKey
     plKeyData* operator->() const;
     plKeyData& operator*() const;
 
-    operator plKeyImp*() const { return (plKeyImp*)fKeyData; }
+    operator bool() const { return fKeyData; }
```

Comment:
Suggestion for clarity:
```suggestion
    operator bool() const { return fKeyData != nullptr; }
```